Header header
bool IsValid
float64 speed_kmh
float64 speed_cms
int8 gear     #档位反馈